apiserver/facades/controller/caasunitprovisioner/provisioner.go

```diff
@@ -217,18 +216,6 @@ func (a *Facade) UpdateApplicationsUnits(args params.UpdateApplicationUnitArgs)
 			result.Results[i].Error = common.ServerError(err)
 			continue
 		}
-
-		appConfig, err := app.ApplicationConfig()
```

Comment:
The intent was to just remove the user facing aspect of this config, not the back end processing. There's logic besides this which would still need to change if the entire config were to be removed.